### PR TITLE
Glicko

### PR DESCRIPTION
**Pull request by Isylwin**

### CODE REVIEW

**Comment by Isylwin** [comment]
retest this please

**Comment by BeeHiveJava** [comment]
retest this please

File: common/src/main/nl/jft/common/glicko/Rating.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+ * <p>
+ * A Glicko-2 rating approaches the skill value of a player through numerical means.
+ * This is done by using a standard deviation and volatility.
+ * The standard deviation represents the certainty about a players rating and the volatility represents the consistency of a player.
+ *
+ * @author Oscar de Leeuw
+ */
+public class Rating {
+
+    private final double rating;
```
Comment (TehMessiah) [inline common/src/main/nl/jft/common/glicko/Rating.java]: MAJOR Rename field "rating" [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1700)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Pull request merged**